test/libService/unittest/ip6tos/ip6tos_test.cpp

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
What happened to the original unit tests? There was a quite comprehensive set at one point, covering all the tricky cases. Could you see if you can resurrect them from the history and incorporate?